mozregression/fetch_configs.py

```diff
@@ -384,38 +402,49 @@ def _common_tk_part(inbound_conf):
 
 
 class FirefoxInboundConfigMixin(InboundConfigMixin):
-    def tk_inbound_route(self, push):
+    def tk_inbound_routes(self, push):
         if self.inbound_branch == 'try' or \
            push.timestamp >= TIMESTAMP_GECKO_V2:
```

Comment:
`TIMESTAMP_GECKO_V2` is in 2015, so I think we can just simplify this chunk to make it unconditional, and remove the buildbot fallback below.

Reply:
I think just to save the rebasing I'll just do a final commit that rips out all the buildbot stuff rather than doing it before.